packages/block-library/src/comment-template/edit.js

```diff
@@ -178,6 +185,8 @@ const MemoizedCommentTemplatePreview = memo( CommentTemplatePreview );
  * @param {Array}  [props.activeComment]    - The block that is currently active.
  * @param {Array}  [props.setActiveComment] - The setter for activeComment.
  * @param {Array}  [props.blocks]           - Array of blocks returned from
+ * @param {Object} [props.firstComment]     - The first comment in the array of
+ *                                          comment objects.
  *                                          getBlocks() in parent .
```

Comment:
Looks like the JSDoc for `[props.blocks]` was torn apart 😅 

```suggestion
 * @param {Array}  [props.blocks]           - Array of blocks returned from
 *                                          getBlocks() in parent .
 * @param {Object} [props.firstComment]     - The first comment in the array of
 *                                          comment objects.
```

Reply:
I've also fixed this 👍